bundles/org.openhab.binding.adorne/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="adorne"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="hub">
+		<label>Adorne Hub</label>
+		<description>The Adorne Hub serves as the bridge to control Adorne switches, dimmer switches and outlets.</description>
+
+		<config-description>
+			<parameter name="host" type="text" required="false">
```

Comment:
```suggestion
			<parameter name="host" type="text">
```

and add `<default>(value that I have forgotten, LCM1.local?)</default>`

same for port

Reply:
Similar to the argument above I would like to keep default handling in the hub controller. And as a user I prefer not to see any value for an optional parameter - if there is a value then I need to think whether it applies to me.

Reply:
So I moved the default handling to AdorneHubConfiguration as per your suggestion. But since host and port are optional parameters I feel the UI should reflect that accordingly. If the user doesn't provide a value then it should come from AdorneHubConfiguration, not the UI.

Reply:
Usually we provide the same value in the configuration and the XML.

Reply:
Ok - made the change for consistency